styles/styles.css

```diff
@@ -0,0 +1,585 @@
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --purple-light: #F2F0FF; /* floating text */
+    --grey: #A7A7A; /* service tile text */
+    --grey-light: #F3F6F9 /* projects background */
+}
+
+/* ------------------------------- 2. Typography ---------------------------*/
+html {
+    font-size: 16px;
+    color: var(--black);
+    font-family: "Roboto", serif;
+    font-weight: 300;
+}
+
+body {
+    margin: 0;
+}
+
+h1 {
+    font-family: "Merriweather", serif;
+    font-weight: 400;
+    text-align: center;
+    font-size: 2.488rem;
+}
+
+.introduction-content h2 {
+    font-family: "Merriweather", serif;
+    text-align: center;
+    font-weight: 500;
+    color: var(--purple);
+    z-index: 1;
+}
+
+.articles h2 {
+    font-family: "Merriweather", serif;
+    font-size: 2.074rem;
+    font-weight: 600;
+}
+
+header {
+    color: white;
+}
+
+a {
+    text-decoration: none;
+    color: white;
+}
+
+.header-subtext {
+    font-size: 1.2rem;
+    text-align: center;
+}
+
+.header-button {
+    font-weight: 100;
+    font-size: 0.694rem;
+    letter-spacing: 1px;
+}
+
+.introduction p {
+    font-weight: 400;
+    text-align: center;
+}
+
+.introduction a {
+    color: var(--purple);
+}
+
+.articles p {
+    font-weight: 400;
+    font-size: 1rem;
+}
+
+.articles a {
+    font-weight: 600;
+}
+
+.squiggle-line {
+    width: 3.333rem;
+    margin-top: -1.111rem;
+}
+
+.project-link {
+    color: var(--purple);
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 0.833rem;
+}
+
+.service h3 {
+    color: var(--purple);
+    font-family: "Merriweather", serif;
+    font-size: 1.44rem;
+    font-weight: 400;
+}
+
+::placeholder {
+    font-family: "Roboto", serif;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+.container {
+    max-width: 78rem;
+    margin: 0 auto;
+    padding: 0 2rem;
+}
+
+.pseudo-background::after {
+    position: absolute;
+    content: "";
+    border-style: solid;
+    border-width: 8vw 0 0 100vw;
+    z-index: 1;
+}
+
+
+
+/*header *,*/
+/*.contact *{*/
+/*    z-index: 2;*/
+/*}*/
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+main {
+    display: flex;
+    flex-direction: column;
+    row-gap: 2.2rem;
+}
+
+/* --------------- 4.1 Header ------------------------- */
+
+header {
+    background: linear-gradient(320deg, rgba(87, 34, 205, 1) 0%, rgba(72, 69, 228, 1) 55%); /* gradient background */
+    position: relative;
+    padding-bottom: 4.5rem;
+    z-index: 0;
+}
+
+/*header::before {*/
```

Comment:
Beste Peter,

Hier zal je de juiste URL moeten toevoegen. De styles.css zit in het mapje styles. Met `..` kan je een stapje lager gaan, dus naar de root waar jou assets mapje staat. 

Hieronder heb ik een verbeterde versie van deze header:

header {
    background: linear-gradient(320deg, rgba(87, 34, 205, 1) 0%, rgba(72, 69, 228, 1) 55%); /* gradient background */
    position: relative;
    padding-bottom: 4.5rem;
    /* z-index: 0; */
}

header::before {
   content: "";
   position: absolute;
   inset: 0 0 0 0;
   width: 100%;
   height: 100%;
   background: url("../assets/noise-texture.png");

Reply:
Ongelofelijk dat het zo'n simpele issue was. Hartstikke bedankt!

Reply:
Geen probleem!